test/.dockerignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
wouldn't the docker image be able to ignore everything under test?

Reply:
And `fastly/`